### ISSUE TITLE
Summarize the question. (e.g., "How to run reconstruction system with RealSense D435i? ")

### ISSUE THREAD

**Issue by aniketghodake10** [isl-org/Open3D#5144]
### Checklist

- [X] I have searched for [similar issues](https://github.com/isl-org/Open3D/issues).
- [X] For Python issues, I have tested with the [latest development wheel](http://www.open3d.org/docs/latest/getting_started.html#development-version-pip).
- [X] I have checked the [release documentation](http://www.open3d.org/docs/release/) and the [latest documentation](http://www.open3d.org/docs/latest/) (for `master` branch).

### My Question

Any idea why TSDF doesn't get properly integrated when we increase stf_trunc value ? Any workaround ?

Basically I want Non-Truncated SDF. So, I tried increasing sdf_trunc parameter. But TSDF seems to be not integrating properly.

Below is the extracted point cloud from same TSDF. We can see very distorted pointcloud. TSDF is also not accurate.

![image](https://user-images.githubusercontent.com/38704659/171362096-8411c628-35be-40f3-81fc-9bd944c3173d.png)
